lib/backend/k8s/conversion/conversion.go

```diff
@@ -207,6 +190,10 @@ func (c Converter) PodToWorkloadEndpoint(pod *kapiv1.Pod) (*model.KVPair, error)
 		}
 	}
 
+	// Add the orchestrator label to the workload endpoint labels.
+	labels := pod.ObjectMeta.Labels
+	labels[apiv2.OrchestratorLabel] = Orchestrator
```

Comment:
I think it would be good to show the Namespace label here as well - that way users can see that the wep has that label on it.